compiler/rustc_codegen_rmc/src/compiler_interface.rs

```diff
@@ -144,11 +145,14 @@ impl CodegenBackend for GotocCodegenBackend {
             None
         };
 
+        let metadata = RmcMetadata { proof_harnesses: c.proof_harnesses };
```

Comment:
Why do we need this?

Reply:
(I'm not sure what you mean by 'this'. The RmcMetadata type?) I'm planning on the `rmc-metadata` file becoming where we emit everything that isn't the symtab. So type_map, function restrictions, and more things like notable/important warnings (instead of just logging them).

Reply:
Sorry, I mean the copy statement.

Reply:
Still not sure what the "copy statement" is.